x-pack/plugins/serverless_search/public/application/components/file_upload_callout.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EuiButton, EuiCallOut, EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import { useKibanaServices } from '../hooks/use_kibana';
+import { FILE_UPLOAD_PATH } from '../constants';
+
+export const FileUploadCallout = () => {
+  const {
+    application: { navigateToUrl },
+    http,
+  } = useKibanaServices();
+  return (
+    <EuiCallOut
+      title={i18n.translate('xpack.serverlessSearch.selectClient.fileUploadCallout.title', {
+        defaultMessage: 'Upload your data from a file',
+      })}
+      size="m"
+    >
+      <p>
+        <FormattedMessage
+          id="xpack.serverlessSearch.selectClient.fileUploadCallout.description"
+          defaultMessage="Upload your file, analyze its data, and import the data into an Elasticsearch index."
+        />
+      </p>
+      <EuiSpacer />
+      <EuiSpacer size="xxl" />
+      <EuiFlexGroup>
+        <EuiFlexItem grow={false}>
+          <EuiButton
+            color="primary"
+            fill
+            data-test-subj="fileupload-callout-cta"
+            onClick={() => navigateToUrl(http.basePath.prepend(FILE_UPLOAD_PATH))}
+          >
```

Comment:
```suggestion
          <EuiButton
            color="primary"
            icon="exportAction"
            data-test-subj="fileupload-callout-cta"
            onClick={() => navigateToUrl(http.basePath.prepend(FILE_UPLOAD_PATH))}
          >
```
We _try_ to have only one primary button on the page. We often fail at that, but in this case, since the two buttons are right next to one another, I'd prefer to make this a non-filled variant and add an icon to the left of the button to illustrate uploading.

Reply:
While we're at it, should we also remove the `fill` from the connectors button? With the fill only on that one it feels like the primary action, and I'm not sure that's what we want.

Reply:
Sound logic @sphilipse - @Samiul-TheSoccerFan Can you implement that suggestion as well?

Reply:
Good point. In the API key section, we have  one primary and one non-filled button however they belong in same panel. @daveyholler any thoughts?